Web/starter_project_vanguard/components/story/Image.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const Image = ({
```

Comment:
is there a syntax to define components in next ? like using React.FC for example?
maybe look into that